.github/workflows/publish-on-release.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish to NPM
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-npm:
+    name: Publish to NPM
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
```

Comment:
Node 14 is the current LTS.

Reply:
Oh, good catch, thanks. Seems we are still using 12 in other places, but good moment to try the upgrade.